### PR TITLE
Readme update & further changes for mergerfs.

### PR DESCRIPTION
**Pull request by Dulanic**
Update for Readme & removal of read_only from plexdrive. read_only was meaninless overall for plexdrive as the final media mount controls that already. Necessary for mergerFS.

### CODE REVIEW

**Comment by Dulanic** [comment]
Updated to make additional changes to allow mergerfs to properly delete.

**Comment by Dulanic** [comment]
Oh I also added a move_ind for people who want to move right away instead of just copying at first. I use this myself to keep stuff off my local drives.

Proof of Test:
image

image

File: config.template [hunk 1]
```diff
@@ -68,7 +68,11 @@ rclone_options="--buffer-size 500M --checkers 16"
 
 rclone_beta="0" # Set this to "1" if you want to live dangerous and try out rclone latest beta.
 
-rclone_mount_options="${rclone_options} --allow-non-empty --allow-other --max-read-ahead 30G"
+if [${pool_choice}] = "1" ]; then
```
Comment (Antexa) [inline config.template]: Typo issue @Dulanic : 

- if [${pool_choice}] = "1" ]; then 
+ if [ "${pool_choice}" = "1" ]; then

Comment (Antexa) [reply config.template]: You should include this modification to the mount.remote script

**Comment by Antexa** [comment]
@Dulanic You really need to test your change before PR, this typo broke the deleting file function

**Pull request merged**